dashboard-frontend/src/types/study.ts

```diff
@@ -3,31 +3,29 @@ export interface Study {
   name: string;
   slug: string;
   createdAt: string;
-  progress: StudyProgress;
-  strata: Strata[];
+  studyProgressList: StudyProgress[];
+  stratumProgressList: StratumProgress[];
 }
 
 interface StudyProgress {
+  id: string;
+  datetime: string;
   currentParticipants: number;
   expectedParticipants: number;
   currentAverageDeviation: number;
   expectedAverageDeviation: number;
-  date: string;
-  averageDeviationOverTime: DataPoint[];
 }
 
-interface Strata {
+interface StratumProgress {
   id: string;
-  percentageDesired: number;
+  name: string;
+  datetime: string;
+  desiredPercentage: number;
   currentPercentage: number;
-  remainingPercentage: number;
   expectedPercentage: number;
-  currentPriceToAdquireParticipant: number;
-  currentBudgetDecided: number;
-  remainingPercentageOverTime: DataPoint[];
-}
-
-interface DataPoint {
-  date: string;
-  value: number;
-}
+  percentageDeviationFromGoal: number;
+  desiredValue: number;
```

Comment:
Why not have this called `currentParticipants` just like in StudyProgress? It's the same value, just per-stratum... Same with `desiredParticipants`...?

Reply:
I thought about it but I ended up using generic names because Stratum seems generic to me.

But you're right it's 0 beneficial to make this generic in the context of an Study. I'll upload the changes now!